### PR TITLE
 [zh-tw]: translate HTML element `<em>`

### PR DESCRIPTION
**Pull request by Dr-XYZ**

### CODE REVIEW

File: files/zh-tw/web/html/element/em/index.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+### \<i> vs. \<em>
+
+某些開發人員可能會對多個元素產生看似相似的視覺結果感到困惑。`<em>` 和 `<i>` 是一個常見的例子，因為它們都使文字變斜。差異在哪裡？應該使用哪個？
+
+默認情況下，視覺結果相同。然而，語義含義是不同的。`<em>` 元素表示其內容的強調重點，而 `<i>` 元素表示與正常散文有所區別的文本，例如外來詞、虛構角色的思想，或當文本指的是詞的定義而不是表示其語義含義時。 （作品的標題，如書籍或電影的名稱，應使用 `<cite>`。）
+
+這意味著使用的正確選擇取決於情況。兩者都不是純粹為了裝飾而使用的，這是 CSS 樣式的用途。
+
+`<em>` 的一個例子可能是：「Just _do_ it already!」或：「We _had_ to do something about it」。閱讀文本的人或軟件將使用口語重音以斜體發音單詞。
```
Comment (yin1999) [inline files/zh-tw/web/html/element/em/index.md:30]: “斜體發音”？这里的翻译是否错误

File: files/zh-tw/web/html/element/em/index.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        <a href="/zh-TW/docs/Web/HTML/Content_categories"
+          >內容類別</a
+        >
+      </th>
+      <td>
+        <a href="/zh-TW/docs/Web/HTML/Content_categories#流內容"
+          >流內容</a
+        >，<a href="/zh-TW/docs/Web/HTML/Content_categories#段落型內容"
+          >段落型內容</a
+        >，可觸及的內容。
```
Comment (yin1999) [inline files/zh-tw/web/html/element/em/index.md:64]:
```suggestion
        >、<a href="/zh-TW/docs/Web/HTML/Content_categories#段落型內容"
          >段落型內容</a
        >、捫及內容。
```

File: files/zh-tw/web/html/element/em/index.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+```html
+<p>
+  Ice cream is made with milk, sweetener, and cream. Frozen custard, on the
+  other hand, is made of milk, cream, sweetener, and <em>egg yolks</em>.
+</p>
+```
+
+### 結果
+
+{{EmbedLiveSample("Example")}}
```
Comment (yin1999) [inline files/zh-tw/web/html/element/em/index.md:47]:
```suggestion
{{EmbedLiveSample("範例")}}
```

**Review by yin1999 — APPROVED** [review]
Thanks @Dr-XYZ

**Comment by yin1999** [comment]
Hey @Dr-XYZ. I guess you have set the wrong email address for git commit: see https://github.com/mdn/translated-content/commit/d2b834fc004a7c2f8705f8e415a54d8a6499cce4.patch

image

**Comment by Dr-XYZ** [comment]
@yin1999  Thanks for reminding.

**Pull request merged**